src/lib/tvgPaint.cpp

```diff
@@ -281,6 +281,14 @@ Result Paint::transform(const Matrix& m) noexcept
     return Result::FailedAllocation;
 }
 
+Matrix Paint::transform() noexcept
+{
+    Matrix m = {1, 0, 0, 0, 1, 0, 0, 0, 1};
```

Comment:
you have two memory allocation.
see below:
auto pTransform = pImpl->transform();
if (pTransform) return *pTransform;
else return {1, 0, 0, 0, 1, 0, 0, 0, 1};